content/applications/general/email_communication/email_domain.rst

```diff
@@ -62,55 +91,64 @@ Enable DKIM
+.. danger::
+   Yahoo or AOL are examples of email providers with a DMARC policy set to ``p=reject``. We
+   strongly advise against using an *@yahoo.com* or *@aol.com* address for your users. These emails
+   will never reach their recipient.
 
-.. seealso:: DMARC official `website <https://dmarc.org/>`_ 
+You can check the DMARC record of a domain name with a tool like
+`MXToolbox DMARC <https://mxtoolbox.com/DMARC.aspx>`_.
 
-If your domain name uses DMARC and has defined one of these policies, it is therefore imperative to be SPF compliant or to enable DKIM.
+If one of your partners, customer or vendor, uses DMARC and has defined one of these policies, the
+Odoo server cannot relay emails from this partner to your users.
+
+You need to :ref:`handle user notifications in Odoo <notification_preferences>`, or replace the email address of the partner with a default email address.
 
-.. danger:: Yahoo or AOL are examples of email providers with a DMARC policy set to ``p=reject``. We strongly advise against using an @yahoo.com or @aol.com address for your users. Using such emails can lead to missing notification, or not receiving emails at all.
+Use a default email address
+===========================
 
-If one of your partners, customer or vendor, uses DMARC and has defined one of these policies, the Odoo server cannot relay emails from this partner to your users. 
+To force the email address from which emails are sent, you need to create one of these two keys in
+the System Parameters of the database:
 
-You need to :ref:`handle user notifications in Odoo <notification_preferences>`, or replace the email address of the partner with a default email address.
+- if ``mail.force.smtp.from`` is set, all outgoing emails are sent from the given address;
+- if ``mail.dynamic.smtp.from`` is set, the from email address is replaced by the given one if the
+  domain does not match the domain set in `mail.catchall.domain`.
 
-SPF, DKIM & DMARC documentation of common providers 
-===================================================
-   
-   - `OVH <https://docs.ovh.com/us/en/domains/web_hosting_how_to_edit_my_dns_zone/>`_
-   - `GoDaddy SPF <https://www.godaddy.com/help/add-a-txt-record-19232>`_
-   - `GoDaddy DKIM <https://www.godaddy.com/help/add-a-cname-record-19236>`_
-   - `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-txtspfdkimdmarc-records-for-my-domain/>`_
-   - `CloudFlare <https://support.cloudflare.com/hc/en-us/articles/360019093151>`_
-   - `Google Domains <https://support.google.com/domains/answer/3290350?hl=en>`_
-   - `Azure DNS <https://docs.microsoft.com/en-us/azure/dns/dns-getstarted-portal>`_
-   - `Mail tester SPF <https://www.mail-tester.com/spf/>`_ (documenting also for other less known providers)
-
-To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give you a full overview of the content and configuration you have in one email sent!
```

Comment:
Added back in as H3 under DMARC H2 since it's related content. 
Slight change: removed Mail-Tester entry from the list and added the expository comment for Mail-Tester into the paragraph. Marking as resolved.

Reply:
Quick follow up to this with something to consider.

Regarding the support article lists in this file, I would simplify these into one condensed list (maybe as a reference section at the end) either as a) a list that only includes one top-level support URL for each provider (to lessen likelihood of broken links / link management on our end) OR b) a tiered list with multiple + specific support articles for each provider.

Option a:

- GoDaddy:
- Namecheap:
- etc.

Option b:

- GoDaddy:
  - SPF
  - DKIM
  - DMARC
- Namecheap:
  - SPF
  - DKIM
  - DMARC
- etc.

I think either of these options would simplify the UX so resources are easier to read + more organized, and may lessen the burden on us to maintain correct links.

What do you think?